website/docs/api/docusaurus.config.js.mdx

```diff
@@ -6,6 +6,12 @@ slug: /api/docusaurus-config
 
 # `docusaurus.config.js`
 
+:::info
+
+Refer to the Getting Started [**Configuration**](docs/configuration.mdx) for examples.
+
+:::
+
```

Comment:
all these things were already in the other PR so shouldn't be there

Reply:
So just this refer sentence and link is good enough here as is now, correct?  You did say you wanted to link back from the API ref to the docs/guide for configuration.  So that is what this info admonition is.